package.json

```diff
@@ -2,8 +2,9 @@
   "name": "material-ui-theme-creator",
   "private": true,
   "description": "A comprehensive tool to customize themes for Material-UI",
-  "version": "4.11.0",
-  "author": "Brian Reynolds <brireyn@gmail.com>",
+  "version": "5.0.0",
+  "original-by": "Brian Reynolds <brireyn@gmail.com>",
+  "v5-author": "Zenoo <https://github.com/Zenoo>",
```

Comment:
Those fields don't exist inside `package.json`.  
Instead, you could use the [`author` and `contributors` fields](https://classic.yarnpkg.com/lang/en/docs/package-json/#toc-maintainers).